lib/chewy/index/adapter/orm.rb

```diff
@@ -124,7 +124,7 @@ def import_objects(collection, options)
             else
               default_scope_where_ids_in(ids)
             end
-
+            batch = batch.select { |object| @options[:searchable_proc].call(object) } if @options[:searchable_proc].present?
```

Comment:
Added searchable proc support - This is used to pass context of valid documents for index while reindexing

Reply:
Existing code from our forked version